.github/actions/test-env-setup/action.yml

```diff
@@ -0,0 +1,46 @@
+name: 'Set up CCBC dev environment'
+description: 'Sets up asdf, perl, and configures the cache'
+runs:
+  using: composite
+  steps:
+    - name: Install asdf
+      uses: asdf-vm/actions/setup@v1
+    - name: Cache asdf Postgres
+      uses: actions/cache@v2
+      env:
+        cache-name: asdf-postgres-cache
+      with:
+        path: |
+          /home/runner/.asdf
+        key: ${{ runner.os }}-postgres-14.1-${{ env.cache-name }}
+        restore-keys: |
+          ${{ runner.os }}-postgres-14.1-${{ env.cache-name }}
+    - name: Setup Postgres DB
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -yqq sqitch libdbd-pg-perl postgresql-client
+        asdf plugin-add postgres || true
+        asdf plugin-update --all
+        asdf install postgres 14.1
+        asdf reshim
+        pg_ctl start
+        createdb ${{ env.PGDATABASE }} || true
+        sqitch rebase -y
```

Comment:
@matthieu-foucault I refactored this but it will currently break without the rebase. At some point we added the rebase in as a fix to a broken deployment as we figured doing a rebase wasn't a huge deal in a CI build.

Reply:
What happens without the rebase? Do you have any error logs you can share?

Reply:
`Cannot find change 5a5afdf81cce0a6b89b18da78f556baf0f655522 (tables/connect_session) in sqitch.plan
Error: Process completed with exit code 2.`

Reply:
I wonder if it would be as simple as renaming the asdf postgres cache, I remember this issue came up when Bryan did a fairly large database rework

Reply:
Right, this was an issue because the job where you created your cache was the same job that deployed the changes, effectively committing those changes in the cache. This shouldn't be an issue now that you have an `install-test-env` job